### PR TITLE
Update examples in docstring to use TF 2.x code

### PR DESCRIPTION
**Pull request by yongtang**
The examples in docstrings of two APIs, tf.histogram_fixed_width_bins
and tf.histogram_fixed_width still used TF 1.x code.

This PR updates the docstring to use TF 2.x code in examples.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Review by alextp — COMMENTED** [review]
Please refer to https://www.tensorflow.org/community/contribute/docs_ref and make these things doctests

**Comment by yongtang** [comment]
Thanks @alextp for the review, the PR has been updated to conform to python doctest. Please take a look and let me know if there are any issues.

**Review by alextp — DISMISSED** [review]

**Review by alextp — DISMISSED** [review]

**Comment by gbaned** [comment]
@yongtang Can you please address Ubuntu Sanity errors? Thanks!

**Review by alextp — DISMISSED** [review]

File: tensorflow/python/ops/histogram_ops.py [hunk 1]
```diff
@@ -62,17 +62,14 @@ def histogram_fixed_width_bins(values,
...
-    variables.global_variables_initializer().run()
-    sess.run(indices) # [0, 0, 1, 2, 4, 4]
-  ```
+  >>> # Bins will be:  (-inf, 1), [1, 2), [2, 3), [3, 4), [4, inf)
+  ...
+  >>> nbins = 5
+  >>> value_range = [0.0, 5.0]
+  >>> new_values = [-1.0, 0.0, 1.5, 2.0, 5.0, 15]
+  >>> indices = tf.histogram_fixed_width_bins(new_values, value_range, nbins=5)
+  >>> print(indices)
```
Comment (alextp) [inline tensorflow/python/ops/histogram_ops.py:71]: Apparently our internal linter is stupid and doesn't like print in comments (it warns about debug prints). Thankfully you can just replace this with `indices` and it'll work just the same.

Can you fix here and in the other print statements?

**Comment by yongtang** [comment]
Thanks @alextp for the help. I changed to `indices.numpy()` as `indices` directly will output more than 80 and pylint may fail (line-too-long). Let me know if this works.

**Review by alextp — APPROVED** [review]

**Pull request merged**